### PR TITLE
Fix swap size check

### PR DESCRIPTION
**Pull request by ololyay**

### CODE REVIEW

File: packages/swap.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 package :swap do
-  swap_size = opts[:swap_size] || 2
+  swap_size = (opts[:swap_size] || 2) * 1024 * 1024
```
Comment (tilast) [inline packages/swap.rb]: oO

**Comment by rilian** [comment]
ok

File: packages/swap.rb (hunk 1 above)
Comment (ololyay) [inline packages/swap.rb]: gb => bytes

Comment (tilast) [inline packages/swap.rb]: ok

**Pull request merged**